docs/devops/create-dotnet-github-action.md

```diff
@@ -184,12 +190,33 @@ The workflow specifies that `on` a `push` to the `main` branch, the action is tr
 
 :::image type="content" source="media/action-log.png" lightbox="media/action-log.png" border="true" alt-text=".NET code metrics - GitHub Action log":::
 
+## Performance improvements
+
+If you followed along the sample, you might have noticed that every time this action is used, it will do a **docker build** for that image. So, every user is faced with some time to build the container before running it. Before releasing your GitHub Action to the marketplace, you should:
```

Comment:
```suggestion
If you followed along the sample, you might have noticed that every time this action is used, it will do a **docker build** for that image. So, every trigger is faced with some time to build the container before running it. Before releasing your GitHub Action to the marketplace, you should:
```